src/components/components/ReinforcementStatement/ReinforcementStatement.scss

```diff
@@ -0,0 +1,52 @@
+.Reinforcement {
+  display: flex;
+  align-items: center;
+  margin: 96px;
+  gap: 96px;
+
+  &__image {
+    border-radius: 8px;
+    max-width: 50%;
+  }
```

Comment:
I think we need extra properties for &__image img {
max-width: 100%;
}
otherwise it will overlap the sibling

Reply:
would you please add it
.Reinforcement__image img {
  max-width: 100%;
}